test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua

```diff
@@ -0,0 +1,137 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1919
+-- Description:
+-- Multiple media and navigation apps activation during active embedded audio source+audio mixing supported
+-- Precondition:
+-- 1) "MixingAudioSupported" = true at .ini file.
+-- 2) SDL and HMI are started.
+-- 3) Three apps running on system:
+--    a. media app_1 in FULL and AUDIBLE
+--    b. navigation app_2 in LIMITED and AUDIBLE
+--    c. non-media app_3 in BACKGROUND and NOT_AUDIBLE
+-- 4) User activates embedded navigation and HMILevel of apps were changed to:
+--    a. media app_1 in LIMITED and AUDIBLE
+--    b. navigation app_2 in BACKGROUND and NOT_AUDIBLE
+-- 5) User activates media app_1 -> FULL, AUDIBLE
+--    a. navigation app_2 still in BACKGROUND and NOT_AUDIBLE
+--    b. non-media app_3 still in BACKGROUND and NOT_AUDIBLE
+-- 6) User activates navigation app_2 -> FULL, AUDIBLE
+-- Steps to reproduce:
+-- 1) User activates non-media app_3
+--    HMI -> SDL: OnAppDeactivated ( <appID_2> )
+--    HMI -> SDL: SDL.ActivateApp ( <appID_3 )
+-- Expected result:
+-- 1) SDL -> navigation app_2 : OnHMIStatus (LIMITED, AUDIBLE)
+-- 2) SDL -> HMI: SDL.ActivateApp (SUCCESS)
+-- 3) SDL -> non-media app_3 : OnHMIStatus (FULL, NOT_AUDIBLE)
+-- Note: media app_1 still in LIMITED and AUDIBLE
+-- Actual result:
+-- 1) SDL does not set required HMILevel and audioStreamingState
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Applications Configuration ]]
+config.application1.registerAppInterfaceParams.appHMIType = { "MEDIA" }
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+config.application2.registerAppInterfaceParams.appHMIType = { "NAVIGATION" }
+config.application2.registerAppInterfaceParams.isMediaApplication = true
```

Comment:
NAVI app should be with isMediaApplication = false

Reply:
@GetmanetsIrina You can find the updates in the commit cd2fa7e